live-examples/js-examples/typedarray-lastindexof.html

```diff
@@ -0,0 +1,7 @@
+<pre>
+<code id="static-js" class="language-js">var uint8 = new Uint8Array([10, 20, 50, 50, 50]);
+
+console.log(uint8.lastIndexOf(50, 4));  // check the first 4 elements
```

Comment:
I think this is checking all 5 elements isn't it? (because it's zero-based, so "4" is actually the last item).

So (I think) that parameter is redundant here.

Again maybe consider using 2 logs here, one omitting the second argument, and one including it (but making it less than array.length):

```
var uint8 = new Uint8Array([10, 20, 50, 50, 50]);

// check all elements
console.log(uint8.lastIndexOf(50, 4));
// expected output: 4

// check the first 4 elements
console.log(uint8.lastIndexOf(50, 3));
// expected output: 3
```